spec/requests/api/v1/articles_spec.rb

```diff
@@ -622,6 +622,8 @@
 
     describe "when authorized" do
```

Comment:
I think it would be nice to split the `article_spec` into separate files like we have in `requests/articles`: `articles_create_spec.rb`, `articles_update_spec.rb` , etc. It's hard to navigate through such a big file and add specs.